svgdigitizer/electrochemistry/electrolyte.py

```diff
@@ -0,0 +1,197 @@
+# ********************************************************************
+#  This file is part of svgdigitizer.
+#
+#        Copyright (C) 2021 Albert Engstfeld
+#        Copyright (C) 2021 Johannes Hermann
+#        Copyright (C) 2021 Julian Rüth
+#        Copyright (C) 2021 Nicolas Hörmann
+#
+#  svgdigitizer is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  svgdigitizer is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with svgdigitizer. If not, see <https://www.gnu.org/licenses/>.
+# ********************************************************************
+from functools import cache
+from pHcalc.pHcalc import Acid, Neutral, System
+from astropy import units as u
+from svgdigitizer.electrochemistry.normalization import normalize_unit
+
+import pubchempy as pcp
+import requests
+import xml.etree.ElementTree as ET
+import re
+
+
+acid_base_dict = {
```

Comment:
```suggestion
acid_bases = {
```
I think that we are usually not encoding the type of a variable in the variable name.